docs/release-notes/8.15.asciidoc

```diff
@@ -1,6 +1,38 @@
 [[release-notes-header-8.15.0]]
 == 8.15
 
+[discrete]
+[[release-notes-8.15.3]]
+=== 8.15.3
+
+[discrete]
+[[known-issue-8.15.3]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[bug-fixes-8.15.3]]
+==== Bug fixes
+* Fixes an {elastic-defend} bug that could cause Linux to crash when {elastic-defend} scans paths (or paths with children) which include virtual file systems, such as procfs.
```

Comment:
Sorry I could have worded this better.  Defend crashes, not Linux.  How about something like this?

```suggestion
* Fixes a bug that could cause {elastic-defend} to crash on Linux when scanning paths (or paths with children) which include virtual file systems, such as procfs.
```